coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -4,7 +4,7 @@ set oem_id="ec2"
 
 # Blacklist the Xen framebuffer module so it doesn't get loaded at boot
 # Disable `ens3` style names, so eth0 is used for both ixgbevf or xen.
-set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never"
+set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc"
```

Comment:
i've seen `tsc=reliable` recommended along with this a couple times, e.g. [this AWS whitepaper](https://d0.awsstatic.com/whitepapers/optimizing-multiplayer-game-server-performance-on-aws.pdf). thoughts?